scripts/common.py

```diff
@@ -654,6 +656,7 @@ def start_redis_server():
         REDIS_SERVER_PATH, REDIS_CONF_PATH,
         '--daemonize', 'yes'
     ])
+    wait_for_port_to_be_open(int(feconf.REDISPORT))
```

Comment:
Why not have feconf.REDISPORT just be an int? There is no point casting it to an int everywhere...

Reply:
The reason is because the port definition that is set to the redis client variable is by default a string. I changed it anyways since the usage of it as an int is more frequent.